worlds/marioland2/options.py

```diff
@@ -0,0 +1,97 @@
+from Options import Toggle, Choice, NamedRange, Range, PerGameCommonOptions
+from dataclasses import dataclass
+
+
+class GoldenCoins(Toggle):
```

Comment:
Any multiline docstrings here should have their closing `"""` on a separate line